projects/1.16/assets/allthemodium/allthemodium/patchouli_books/allthemodium/zh_cn/entries/links/github.json

```
@@ -0,0 +1,12 @@
+{
+    "name": "Github",
+    "icon": "allthemodium:unobtainium_block",
+    "category": "allthemodium:links",
+    "sortnum": "1",
+    "pages": [{
+        "type": "link",
+        "url": "https://github.com/AllTheMods",
+        "link_text": "GitHub 链接",
+        "text": "在 GitHub 中，你可以查看当前的问题和整合包更新。"

```

```suggestion
        "text": "在 GitHub 中，你可以查看当前的议题和整合包更新信息。"
```
```suggestion
        "text": "在 GitHub 中，你可以查看当前的 issue 和整合包更新信息。"
```
issue在github自己的文档里都有`问题`和`议题`两种叫法（）